src/hammer-tech/schema.json

```diff
@@ -194,7 +194,64 @@
             "type" : "string"
           }
         },
-        "additionalProperties" : false
+        "additionalproperties" : false
+      }
+    },
+    "stackups" : {
```

Comment:
Does this make the old metal layers (https://github.com/ucb-bar/hammer/pull/308/files#diff-57aef1ccd100a4823016fa230d58d5e7L77) defunct?
If so we should remove that.

Reply:
oh, huh. Yes, I think so.

Reply:
As discussed in meeting this should be dead and will be removed.

Reply:
@edwardcwang It appears this is unused in hammer. It's used in icc and dc... what are the plans for those plugins? I don't want to spend a lot of time "correcting" them if they're broken to begin with.

Reply:
I commented them out of those tools, so it behaves as if the field is set to `None` There is a link to this PR in the comments.